### PR TITLE
Fix small typos in How-to-Create-and-Maintain-a-Tap.md

### PR DESCRIPTION
**Pull request by kareman**

### CODE REVIEW

File: docs/How-to-Create-and-Maintain-a-Tap.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ avoid conflicts.
 
 ## Maintaining a tap
 A tap is just a git repository so you don’t have to do anything specific when
-making modifications apart from committing and pushing your changes.
+making modifications,  apart from committing and pushing your changes.
```
Comment (MikeMcQuaid) [inline docs/How-to-Create-and-Maintain-a-Tap.md]: Only need one space here.

**Comment by kareman** [comment]
Sorry about that, fixed it now.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by kareman** [comment]
Why thank you. You're not so bad yourself :)

**Pull request merged**